README.md

```diff
@@ -1 +1,20 @@
- -
+# CocoaHeads Russia
+
+We are the biggest community of iOS/macOS Developers in Russia. We organize developers meetups in Moscow and St.Petersburg.
+
+Follow us on [Twitter](https://twitter.com/cocoaheadsru) and [Facebook](https://www.facebook.com/cocoaheads) or join us in [Telegram](https://t.me/cocoaheads)!
```

Comment:
Shouldn't we mention VK group and YouTube channel?

Reply:
Fixed